ch03.asciidoc

```diff
@@ -524,24 +524,37 @@ As payments are onion-routed, the users who route the payment will only be aware
 This is because the security model of Lightning only requires users to be aware of the balances in their channels.
 They are not required to be aware of the state of the entire network.
 
-==== Waiting for Confirmations vs Instant settlement
+==== Waiting for Confirmations vs Instant Settlement
 
 On the Bitcoin network, transactions are only 'settled' once they have been included in a block, in which case they are 'confirmed' in that block.
-As more blocks are mined on the Bitcoin network after this, the transaction acquires more 'confirmations' and is considered more secure.
+As more blocks are mined on the Bitcoin network thereafter, the transaction acquires more 'confirmations' and is considered more secure.
 
-On the Lightning Network, confirmation only matters for opening and closing channels.
+On the Lightning Network, confirmations only matters for opening and closing channels on-chain.
 Once a Funding Transaction has reached a suitable number of confirmations (e.g. 3), the channel partners consider the channel open.
-As the bitcoin in the channel is secured by the smart contract that manages that channel, payments settle instantly once received by the receiver and are not reversible.
+As the bitcoin in the channel is secured by the smart contract that manages that channel, payments settle instantly once received by the receiver.
+In practical terms, instant settlement means that payments take on the order of 1 second to execute.
```

Comment:
I'd say relax this to "on the order of a few seconds". EMV chips as is today can at times take up to 10 seconds to "complete" at physical PoS system, which people still perceive to be "instant".

Reply:
Totally agree. We should not promise too much and always keep it realistic and factual.

Thanks for catching it.

Commit # 2 in c415d178ba8797d0f05396a87800dc234103e33d